Assets/VRTK/SDK/HyperealVR/SDK_HyperealVRBoundaries.cs

```diff
@@ -0,0 +1,111 @@
+﻿// HyperealVR Boundaries|SDK_HyperealVR|005
+namespace VRTK
+{
+#if VRTK_DEFINE_SDK_HYPEREALVR
+    using UnityEngine;
+    using Hypereal;
+#endif
+
+    /// <summary>
+    /// The HyperealVR Boundaries SDK script provides a bridge to the HyperealVR SDK play area.
+    /// </summary>
+    [SDK_Description(typeof(SDK_HyperealVRSystem))]
+    public class SDK_HyperealVRBoundaries
+#if VRTK_DEFINE_SDK_HYPEREALVR
+        : SDK_BaseBoundaries
+#else
+        : SDK_FallbackBoundaries
+#endif
+    {
+#if VRTK_DEFINE_SDK_HYPEREALVR
+        protected HyRender cachedHyperealVRPlayArea;
+        private const float thickness = 0.1f;
+
+        /// <summary>
+        /// The InitBoundaries method is run on start of scene and can be used to initialse anything on game start.
+        /// </summary>
+        public override void InitBoundaries()
+        {
+        }
+
+        /// <summary>
+        /// The GetPlayArea method returns the Transform of the object that is used to represent the play area in the scene.
+        /// </summary>
+        /// <returns>A transform of the object representing the play area in the scene.</returns>
+        public override Transform GetPlayArea()
+        {
+            cachedPlayArea = GetSDKManagerPlayArea();
+            if (cachedPlayArea == null)
+            {
+                GameObject myHeadGO = VRTK_SharedMethods.FindEvenInactiveGameObject<HyHead>();
+                cachedPlayArea = myHeadGO != null ? myHeadGO.transform.parent : null;
```

Comment:
wrap this in brackets to make it consistent with other uses of this pattern.

`cachedPlayArea = (myHeadGO != null ? myHeadGO.transform.parent : null);`